Makefile

```diff
@@ -0,0 +1,12 @@
+test: install
+	go test ./...
+
+check: test
+	@echo Checking code is gofmted
```

Comment:
It seems like we will be repeating the same sets of build checks.  Perhaps we could rally on a single script.